org.sasview.sasview.yml

```diff
@@ -0,0 +1,58 @@
+id: org.sasview.sasview
+runtime: org.kde.Sdk
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 6.7
```

Comment:
if runtime is 5.15-23.08, this should be 5.15-23.08 too? This branch is based off Qt6

Reply:
or is the runtime wrong? Because you have pyside6 6.8 in python sources, that needs Qt6

Reply:
It runs fine with the current version but I also see no reason not to just upgrade the runtime. I'll just do that.

Reply:
Upgrading the runtime to 6.7 works.